RecoTauTag/HLTProducers/python/applyL2TauTag.py

```diff
@@ -7,7 +7,8 @@ def insertL2TauSequence(process, path, ref_module):
     ref_idx = path.index(ref_module)
     path.insert(ref_idx + 1, process.hltL2TauTagNNSequence)
     path.insert(ref_idx + 2, process.hltL2DoubleTauTagNNFilter)
-    path.insert(ref_idx + 3, process.HLTGlobalPFTauHPSSequence)
+    if process.HLTGlobalPFTauHPSSequence not in process.paths:
```

Comment:
this is wrong, in case if `path._seq` does not work, better to remove that `if` for the moment.